x-pack/platform/plugins/shared/fleet/server/services/package_policy.ts

```diff
@@ -1586,12 +1759,14 @@ class PackagePolicyClientImpl implements PackagePolicyClient {
 
     const secretsToDelete: string[] = [];
     if (idsToDelete.length > 0) {
-      const { statuses } = await soClient.bulkDelete(
-        idsToDelete.map((id) => ({ id, type: savedObjectType })),
-        {
-          force: true, // need to delete through multiple space
-        }
-      );
+      const { statuses } = await soClient
+        .bulkDelete(
```

Comment:
I only changed the code here to add the `catch()`.... But I wonder if this actually works when using an un-scoped SO client, since what I been experiencing is that the SO client methods that update/create may need a properly scoped client.